ci/gpu/build.sh

```diff
@@ -60,14 +60,20 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c rapidsai/label/x
       "dask-cuda=${MINOR_VERSION}" \
       "ucx-py=0.12*" \
       "statsmodels" \
-      "xgboost=0.90.rapidsdev1"
+      "rapids-xgboost>=0.12"
```

Comment:
@dantegd is this what you had in mind? Still conda installing even though we pip install below? That looks so weird to me!!

Reply:
This basically forces it to pull all of the dependencies from conda and so when it does the pip install below it finds the conda packages instead of using pip to install dependencies.

Reply:
```suggestion
      "rapids-xgboost>=0.13"
```

Reply:
There is no 0.13 build yet for rapids-xgboost.

Reply:
there is according to the conda repo: https://anaconda.org/rapidsai-nightly/rapids-xgboost/files